lectures/monte_carlo.md

```diff
@@ -485,23 +485,23 @@ Here $\{\eta_t\}$ is also IID and standard normal.
 For the dynamic model, we adopt the following parameter values.
 
 ```{code-cell} ipython3
-μ  = 0.0001
-ρ  = 0.1
-ν  = 0.001
-S0 = 10
-h0 = 0
+default_μ  = 0.0001
+default_ρ  = 0.1
+default_ν  = 0.001
+default_S0 = 10
+default_h0 = 0
 ```
```

Comment:
@longye-tian I wonder if this would be better setup as a `named tuple` called `defaults` so that would then be referenced as `defaults.μ` as a tidier solution.